.gitlab-ci.yml

```diff
@@ -545,7 +545,7 @@ build/cuda110/intel/cuda/debug/static:
 # HIP AMD
 build/amd/gcc/hip/debug/shared:
   <<: *default_build_with_test
-  image: localhost:5000/gko-amd-gnu8-llvm7
+  image: localhost:5000/intel_dgpu_6ee821ee_gko-amd-gnu8-llvm7
```

Comment:
Do we have a common container for Intel DPCPP builds and HIP builds?

Reply:
There is no such thing. It's only that as a side effect of the containers being regenerated for the new DPC++ CI, ROCm updated as well and everything broke (pretty much).